app/pb_kits/playbook/pb_project/_project.scss

```diff
@@ -0,0 +1,7 @@
+[class^=pb_project_kit] {
+  .links{
+    span{
```

Comment:
Can you make `links` selector more unique? Perhaps `pb_project_kit_links`? Can you add a space between the selector and the curly brace?

Reply:
Good catch! Thanks!
Changes here 2a431c6